plugins/transport-grpc/build.gradle

```diff
@@ -0,0 +1,197 @@
+import org.gradle.api.attributes.java.TargetJvmEnvironment
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+plugins {
+  id('com.google.protobuf') version 'latest.release'
+}
```

Comment:
Without the proto files I don't think you need this or the whole `protobuf {` block starting on line 42.